src/Library/Lod/LodVersion.h

```diff
@@ -7,5 +7,3 @@ enum class LodVersion {
     LOD_VERSION_MM7,
     LOD_VERSION_MM8,
 };
-
-using enum LodVersion;
```

Comment:
Please keep this one, it makes it possible to refer to `LodVersion` values w/o the `LodVersion::` prefix. Having to type out `LodVersion::LOD_VERSION_MM6` every time is tedious.